.github/workflows/testing-integration.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           DEBUG: ${{ matrix.debug }}
           SKIP_TESTS_CLEANUP: ${{ matrix.coverage }}
           SUITES: wpunit
+          COMPOSER_MEMORY_LIMIT: "-1"
```

Comment:
♾️

Reply:
ya, trying to overcome some memory limit issues, but going to make that a separate issue